server/index.js

```diff
@@ -2,16 +2,17 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const cors = require('cors');
 const app = express();
+const db = require('mysql');
 
-const server = require('http').Server(app)
-exports.server = server
+const server = require('http').Server(app);
+exports.server = server;
 
-require('./sockets')
+require('./sockets');
 
 app.use(bodyParser.json());
 app.use(cors());
 
-const PORT = process.env.PORT || 1010
+const PORT = process.env.PORT || 1010;
```

Comment:
Why are we editing this class in general?

Reply:
that was some stuff that was accidentally  touched up

Reply:
Okie dokie, that's fine.